commands/get.js

```diff
@@ -1,6 +1,51 @@
 import { resolve, join, posix, dirname, basename } from 'node:path'
-import { writeFileSync, statSync, existsSync, mkdirSync } from 'node:fs'
+import { statSync, existsSync, mkdirSync } from 'node:fs'
 import { connect } from '@existdb/node-exist'
+import { writeFile } from 'node:fs/promises'
+
+let inflightPromises = 0
+
+const queue = []
+
+async function handleNextInQueue () {
```

Comment:
Consider aligning with the solution in download command.

Either remove the dependency on Bottleneck entirely or use it here as well